src/aleph/schemas/api/accounts.py

```diff
@@ -69,10 +74,31 @@ class AddressBalanceResponse(BaseModel):
     model_config = ConfigDict(from_attributes=True)
 
     address: str
-    balance: str
+    balance: FloatDecimal
     chain: Chain
 
 
+class GetCreditBalancesQueryParams(BaseModel):
+    pagination: int = Field(
+        default=100,
+        ge=0,
+        description="Maximum number of credit balances to return. Specifying 0 removes this limit.",
+    )
+    page: int = Field(
+        default=DEFAULT_PAGE, ge=1, description="Offset in pages. Starts at 1."
+    )
+    min_balance: int = Field(
+        default=0, ge=1, description="Minimum Credit Balance needed"
+    )
+
+
+class AddressCreditBalanceResponse(BaseModel):
+    model_config = ConfigDict(from_attributes=True)
+
+    address: str
+    credits: FloatDecimal
```

Comment:
I think that we agreed to not allow decimals on credits.

Reply:
It is an open door just in case we change our mind on the while. For now we are always sending integer values on the credit distribution message, so there won't be any decimal value anywhere for credits